### ISSUE TITLE
Configuration change

### ISSUE THREAD

**Issue by orikl84** [MindorksOpenSource/android-mvp-architecture#31]
How do you save the presenters state during a configuration change?

**Comment by amitshekhariitbhu** [comment]
Yes, We need to update the project for this. Very soon, this will be in the project.

**Comment by Wakirin** [comment]
It's updated ?

**Comment by testacnt145** [comment]
No update yet

**Comment by VasylT** [comment]
Hi. Have you implemented 'Config change' at least in one of your arhitecture sample? It's crucial part of development, but suprisingly everyone skips it in examples:)

**Comment by amitshekhariitbhu** [comment]
We have implemented this in the mvvm architecture.
